scripts/index.js

```diff
@@ -15,6 +15,11 @@ if (appConfig.features.useTansaProofing) {
     require('apps/tansa');
 }
 
+// load locale for angular
+if (appConfig.ddangular_i18n) {
+	require('angular-i18n/' + appConfig.angular_i18n);
```

Comment:
Dynamic imports are not possible with webpack (or any other bundler as far as I know). If you do this, everything inside that folder will be imported at compile time and the correct import will be selected at runtime. It's a bit much to include this many files in the project. We have to find a better way to do this.